### ISSUE TITLE
How to handle dynamic download link to download a file?

### ISSUE THREAD

**Issue by VignanTej** [Steveorevo/node-red-contrib-nbrowser#23]
A big thumbs-up to the contributers for this amazing node. Makes life a lot easier.

I am not able to understand how to download a file which gets downloaded after clicking the element which is formatted as follows.

`<div ng-show="orderHistory.orders.length > 0" class="order-csv" ng-click="exportPastOrdersData()" style="">
                                <h4>Export as CSV</h4>
                                <i class="glyphicon glyphicon-cloud-download"></i>
                                <span ng-show="csvFailed" class="hide ng-hide">CSV download failed, Please try again.</span>
                            </div>`

This would, after 4-5 seconds start the download in a new pop up window. How can save the downloaded csv?

**Comment by Steveorevo** [comment]
Did you fill out the onDownload method? From the documentation:


`onDownload`
Use this method to save a downloaded item from the web. Be sure to add this method before the gotoURL method or prior to invoking a download link.
`file` - The path and filename of the item to be downloaded.
`output` - An existing context property to return the download results to or flow output (default). When using a flow output, the download results will be in msg.payload.

**Comment by VignanTej** [comment]
Yes I tried that. It is not a direct url or link. It runs a JavaScript and
opens a new window for the download link.
I need to download the file opened in the new window.

**Comment by Steveorevo** [comment]
Your best bet is to determine the URL it is attempting to open in a new window and use gotoURL, after the onDownload method.
